src/bucket/BucketList.cpp

```diff
@@ -510,16 +565,98 @@ BucketList::getSize() const
 }
 
 void
-BucketList::addBatch(Application& app, uint32_t currLedger,
-                     uint32_t currLedgerProtocol,
-                     std::vector<LedgerEntry> const& initEntries,
-                     std::vector<LedgerEntry> const& liveEntries,
-                     std::vector<LedgerKey> const& deadEntries)
+HotArchiveBucketList::addBatch(Application& app, uint32_t currLedger,
+                               uint32_t currLedgerProtocol,
+                               std::vector<LedgerEntry> const& archiveEntries,
+                               std::vector<LedgerKey> const& restoredEntries,
+                               std::vector<LedgerKey> const& deletedEntries)
+{
+    ZoneScoped;
+    releaseAssert(currLedger > 0);
+
+    for (uint32_t i = static_cast<uint32>(mLevels.size()) - 1; i != 0; --i)
+    {
+        /*
```

Comment:
Don't forget to remove this (or uncomment if that's universally useful for debuggin)